UI/lib/ui-header.html

```diff
@@ -23,7 +23,6 @@
         # *persistent*
         # http://www.w3.org/TR/html401/present/styles.html#h-14.3.1
     -%]
-    <%= htmlWebpackPlugin.tags.headTags %>
```

Comment:
Let the headTags there, remove the bodyTags below and assign the HtmlWebpackPlugin `inject` parameter to 'body' instead of false in webpack.config.js.
This will let HtmlWebpackPlugin manage assets and load them in the desired section

Reply:
Just checked. The effect is that the scripts are injected *after* the closing `</html>` tag. Are we Ok with that?

Reply:
It does load the css in head, scripts after body and shows the desired behaviour, no?

Reply:
Absolutely. I'll PR.